src/test/java/com/vytrack/tests/US64_FilterCustomersInfoOnAccountPage.java

```diff
@@ -0,0 +1,170 @@
+
+            Actions actions = new Actions(Driver.getDriver());
+
+
+            //Step2. Hover over to Customer module
+            BrowserUtils.sleep(2);
+            actions.moveToElement(customerModule).perform();
+
+            //click Account option
+            WebElement AccountsOption = Driver.getDriver().findElement(By.xpath("//span[.='Accounts']"));
+            AccountsOption.click();
+            BrowserUtils.sleep(2);
+
+            //Step3.locating actual Options
+            WebElement fistModule = Driver.getDriver().findElement(By.xpath("(//span[@class='grid-header-cell__label'])[1]"));
+            String fistModuleText = fistModule.getText();
+
+            WebElement secondModule = Driver.getDriver().findElement(By.xpath("(//span[@class='grid-header-cell__label'])[2]"));
+            String secondModuleText = secondModule.getText();
+
+            WebElement thirdModule = Driver.getDriver().findElement(By.xpath("(//span[@class='grid-header-cell__label'])[3]"));
+            String thirdModuleText = thirdModule.getText();
+
+            WebElement forthModule = Driver.getDriver().findElement(By.xpath("(//span[@class='grid-header-cell__label'])[4]"));
+            String fourthModuleText = forthModule.getText();
+
+            WebElement fifthModule = Driver.getDriver().findElement(By.xpath("(//span[@class='grid-header-cell__label'])[5]"));
+            String fifthModuleText = fifthModule.getText();
+
+            WebElement sixthModule = Driver.getDriver().findElement(By.xpath("(//span[@class='grid-header-cell__label'])[6]"));
+            String sixthModuleText = sixthModule.getText();
+
+            WebElement seventhModule = Driver.getDriver().findElement(By.xpath("(//span[@class='grid-header-cell__label'])[7]"));
+            String seventhModuleText = seventhModule.getText();
+
+            List<String> actualModuleTexts = new ArrayList<>(Arrays.asList(fistModuleText, secondModuleText, thirdModuleText, fourthModuleText, fifthModuleText, sixthModuleText, seventhModuleText));
+            System.out.println("actualModuleTexts = " + actualModuleTexts);
+
+            //Verify actual module texts with expacted module texts
+            List<String> expectedModuleTexts = new ArrayList<>(Arrays.asList(
+                    "Account Name",
+                    "Contact Name",
+                    "Contact Email",
+                    "Contact Phone",
+                    "Owner",
+                    "Business Unit",
+                    "Created At",
+                    "Updated At"));
+
+            Assert.assertEquals(actualModuleTexts, expectedModuleTexts);
+
+        }
+
+        //testing modules for stores manager
+        @Test
+        public void filteringCustomersInfoForStoreManager() {
+            //Step1.login as Store Manager
+            VytrackUtils.loginAsStoreManger();
+            WebElement customerModule = Driver.getDriver().findElement(By.xpath("(//span[@class='title title-level-1'])[3]"));
```

Comment:
I would find element using name not index.
For example "//span[normalize-space()='Customers' and contains(@class='title title-level-1')]"

Reply:
Lilya I'm not sure can you please ask David Please?

Reply:
Lilya Can you please ask someone from like Adam?

Reply:
Actually I have no question. It was just recommendation.